Dockerfile

```diff
@@ -3,9 +3,6 @@ FROM google/cloud-sdk:284.0.0-alpine
 ARG BUILD_DATE
 ARG VCS_REF
 
-RUN echo $BUILD_DATE
```

Comment:
@primeroz I'm assuming this was a typo (so removed), but wanted to check.

I believe this was due to debugging...

Reply:
yeah, probably just a check ?